packages/.vscode/settings.json

```diff
@@ -2,4 +2,15 @@
     // Set current working directory to devtools_app.
     "terminal.integrated.cwd": "devtools_app",
     "dart.showTodos": false,
+    "cSpell.words": [
+        "autofocus",
+        "discoverability",
+        "gcing",
+        "kenz",
+        "listenables",
+        "peterdjlee",
+        "snapshotting",
+        "unawaited",
+        "unfocus"
+    ],
```

Comment:
Let's remove this from this PR. I tried to add this cSpell VS code extension and the list of exceptions we would need to add for our repo was large and continued to grow. If you want to add this to your local workspace settings, that is fine but I don't think we should add this at the repo level.